npm/cypress-schematic/src/e2e.spec.ts

```diff
@@ -26,16 +26,16 @@ const cypressSchematicPackagePath = path.join(__dirname, '..')
 
 const ANGULAR_PROJECTS: ProjectFixtureDir[] = ['angular-13', 'angular-14']
 
-describe('cypress-schematic-e2e', function () {
+describe('ng add @cypress/schematic / only e2e', function () {
   this.timeout(1000 * 60 * 4)
 
   for (const project of ANGULAR_PROJECTS) {
-    it('should', async () => {
+    it('should install e2e files by default', async () => {
       const projectPath = await scaffoldAngularProject(project)
 
       await runCommandInProject(`yarn add @cypress/schematic@file:${cypressSchematicPackagePath}`, projectPath)
-      await runCommandInProject('yarn ng add @cypress/schematic --e2eUpdate', projectPath)
-      await runCommandInProject('yarn ng e2e angular --watch false', projectPath)
+      await runCommandInProject('yarn ng add @cypress/schematic --e2e --ct false --add-ct-specs false', projectPath)
```

Comment:
The `by default` would mean we don't need the extra flags so `yarn ng add @cypress/schematic` should be sufficient